gratipay/utils/fake_data.py

```diff
@@ -167,6 +167,22 @@ def fake_elsewhere(db, participant, platform):
                , extra_info=None
                 )
 
+def fake_payment(db, participant, team):
+    """Create fake payment
+    """
+    direction = ['to-team','to-participant']
+    if participant.username == team.owner:
+        direction = 'to-participant'
+    else:
+        direction = random.sample(['to-team','to-participant'],1)[0]
```

Comment:
Also, since teams don't carry a balance - we need the net payments to sum up to zero. I've added a db check for this in https://github.com/gratipay/gratipay.com/pull/3597 :) Maybe try rebasing on that to make sure?